ChangeLog.d/fix-tls-padbuf-zeroization

```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * Fix a case where potentially sensitive information would not be completely
+     zeroized during TLS handshake.
```

Comment:
Can we be more specific about the affected scenarios? E.g., is it 1.2 only, is it only for certain key exchanges, etc etc?

Reply:
I have tried to be specific as I can be - yes this is TLS1.2 only, but its not dependant on hash used, all paths lead to this.